### PR TITLE
chore(url): allow symbols and pictographs in url

### PR DESCRIPTION
**Pull request by prousso**

### CODE REVIEW

File: src/validators/url.py [hunk 1]
```diff
@@ -29,6 +29,8 @@ def _path_regex():
     return re.compile(
         # allowed symbols
         r"^[\/a-z0-9\-\.\_\~\!\$\&\'\(\)\*\+\,\;\=\:\@\%"
+        # symbols / pictographs
+        + r"\U0001F300-\U0001F5FF"
         # emoticons / emoji
         + r"\U0001F600-\U0001F64F"
```
Comment (yozachar) [inline src/validators/url.py:35]: These ranges overlap. You could simply have:

```diff
- # symbols / pictographs
- r"\U0001F300-\U0001F5FF"
- # emoticons / emoji
- r"\U0001F600-\U0001F64F"
+ # symbols / pictographs / emojis / emoticons
+ r"\U0001F300-\U0001F5FF"
```

**Review by yozachar — CHANGES_REQUESTED** [review]

File: src/validators/url.py (hunk 1 above)
Comment (yozachar) [reply src/validators/url.py:35]: Also, please rebase `prousso:chore/add-symbols-pictographs-support` on `master`, when you're done.

Comment (prousso) [reply src/validators/url.py:35]: Do you mean
```suggestion
+ # symbols / pictographs / emojis / emoticons
+ + r"\U0001F300-\U0001F64F"
```
?

Comment (yozachar) [reply src/validators/url.py:35]: Sorry, my bad keep them separate.

ref: https://unicodeplus.com/plane/1

**Review by yozachar — APPROVED** [review]

**Comment by yozachar** [comment]
Thanks!

**Comment by prousso** [comment]
Could it be possible to release it ?

**Pull request merged**